.cicd/generate-pipeline.sh

```diff
@@ -693,7 +693,11 @@ cat <<EOF
   - label: ":beer: Brew Updater"
     command:
       - "buildkite-agent artifact download eosio.rb . --step ':darwin: macOS 10.14 - Package Builder'"
-      - "buildkite-agent artifact upload eosio.rb"
+      - "mv eosio.rb eosio_mojave.rb
+      - "buildkite-agent artifact upload eosio_mojave.rb"
+      - "buildkite-agent artifact download eosio.rb . --step ':darwin: macOS 10.15 - Package Builder'"
+      - "mv eosio.rb eosio_catalina.rb
+      - "buildkite-agent artifact upload eosio_catalina.rb"
```

Comment:
You can get rid of this list syntax quoting nonsense, if you want to. It is only necessary for the Anka steps and, frankly, seems very error-prone because of the quotes.
```yaml
command: |
  buildkite-agent artifact download eosio.rb . --step ':darwin: macOS 10.14 - Package Builder'
  mv eosio.rb eosio_mojave.rb
  buildkite-agent artifact upload eosio_mojave.rb
  buildkite-agent artifact download eosio.rb . --step ':darwin: macOS 10.15 - Package Builder'
  mv eosio.rb eosio_catalina.rb
  buildkite-agent artifact upload eosio_catalina.rb
```